webtraversallibrary/processtools.py

```diff
@@ -47,6 +53,19 @@ def __get__(self, inst, cls):
         return result
 
 
+class Alarm(Thread):
+    """Helper class to run a timeout thread on Windows"""
+
+    def __init__(self, timeout):
+        Thread.__init__(self)
+        self.timeout = timeout
+        self.setDaemon(True)
+
+    def run(self):
+        sleep(self.timeout)
+        os._exit(1)  # pylint: disable=protected-access
```

Comment:
Is the normal `exit(1)` not applicable here? Or is it because it's on its own thread?

Reply:
There's a difference - this one is used for forked threads.

Reply:
I see, that's what I assumed :)